test/acceptance/web-service.js

```diff
@@ -99,7 +99,7 @@ describe('web-service', function(){
       request(app)
       .get('/api/something?api-key=bar')
       .expect('Content-Type', /json/)
-      .expect(404, '{"error":"Lame, can\'t find that"}', done)
+      .expect(404, '{"error":"Sorry, can\'t find that"}', done)
```

Comment:
nit pick: while you are in this bit we could use backticks to remove the need to escape
```
 .expect(404, `{"error":"Sorry, can't find that"}`, done)
```

Reply:
Not a nitpick! Thanks!

Reply:
That didn't work.

    npm run lint
    > express@4.17.1 lint C:\projects\express
    > eslint .
    C:\projects\express\test\acceptance\web-service.js
      102:20  error  Parsing error: Unexpected character '`'
    ✖ 1 problem (1 error, 0 warnings)

Reverting.

Reply:
I suspect the eslint config is not configured to support template literals https://eslint.org/docs/rules/prefer-template re https://eslint.org/docs/user-guide/configuring#specifying-environments.  I guess changing that is a bigger project wide decision.